types/entities.go

```diff
@@ -16,10 +16,30 @@ type Entities map[EntityUID]*Entity
 // An Entity defines the parents and attributes for an EntityUID.
 type Entity struct {
 	UID        EntityUID   `json:"uid"`
-	Parents    []EntityUID `json:"parents,omitempty"`
+	Parents    []EntityUID `json:"parents"`
```

Comment:
The Cedar [docs](https://docs.cedarpolicy.com/auth/entities-syntax.html#entities) say that `parents` should always be present:

> At the top level, Cedar expects a JSON list (an array using [ ]) of objects. Each object in this list represents a single entity, and should have three attributes:

[uid](https://docs.cedarpolicy.com/auth/entities-syntax.html#uid)
[parents](https://docs.cedarpolicy.com/auth/entities-syntax.html#parents)
[attrs](https://docs.cedarpolicy.com/auth/entities-syntax.html#attrs)